pycronofy/client.py

```diff
@@ -964,3 +964,24 @@ def hmac_valid(self, hmac_string, body):
 
         # compare_digest used to reduce vulnerability to timing attacks
         return any(hmac.compare_digest(value.encode(), calculated) for value in hmac_list)
+
+    def get_ui_element_token(self, permissions, subs, origin, version="1"):
+        """Get a UI element token for rendering UI elements
+
+        :param permissions: List of strings containing the permissions required.
+            See documentation for info: https://docs.cronofy.com/developers/ui-elements/authentication/#param-permissions
+        :param subs: List of strings containing the subs of the users to include in the
+            element token request
+        :param origin: String containing the HTTP origin that the element token will be
+            used to render elements on
+        :param version: Version of the element token to request
+        """
+        return self.request_handler.post(
+            endpoint="element_tokens",
+            data={
+                "permissions": permissions,
+                "subs": subs,
+                "origin": origin,
+                "version": version
+            }
+        ).json()["token"]
```

Comment:
It would be good to return the whole response dict in this case, as it contains an `expires_in` value which some consumers may wish to access

Reply:
Yep, makes sense. I've updated this now.